_includes/feedback.html

```diff
@@ -0,0 +1,8 @@
+<br/>
+<div class="whats-new-post" style="padding: 15px; background: #e4f0f9; border-left: 2px solid blue;">
+<h4 style="margin-top: 5px"> Give us your feedback! </h4>
+<p>
+	Every quarter OpenSearch obtains feedback from users. This helps us make improvements to the OpenSearch Project. Please take a few minutes to take this survey.
+</p>
+<a class="cta" style="padding: 10px; border-radius: 5px; border: 1px solid blue;" href="https://amazonmr.au1.qualtrics.com/jfe/form/SV_1BxJNrtCo4LbweW">Take the survey.</a>
```

Comment:
Lets set the bottom margin for the CTA button to be 0px in this case. Right now its got way too much space underneath. That should help make it fit in better.

Reply:
Also, lets chance the text on the button to "Take the Survey". That way we are matching the style of our other buttons better.